openpaas-james/mailbox/encrypted/api/src/test/scala/com/linagora/openpaas/encrypted/KeystoreManagerContract.scala

```diff
@@ -8,6 +8,7 @@ import reactor.core.scala.publisher.{SFlux, SMono}
 
 import scala.jdk.CollectionConverters._
 
+// Remark: why is this not in test folder? Need to move =>
```

Comment:
Rebase on master?

(and remove this comment?)

Reply:
Pretty sure I did remove it in a fixup, probably came back when I rebased... Thanks for spotting this

Reply:
and the PR was already rebased on master :)